src/index.html

```diff
@@ -1,13 +1,453 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link
+      rel="icon"
+      href="./images/Met/icon.png"
+      type="TheMet-icon.png"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body .page__body--with-menu">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-action">
+            <a href="/" class="logo"></a>
```

Comment:
Checklist 5th and 6th items, fix it everywhere
![image](https://user-images.githubusercontent.com/37161315/170654905-ca16ba2b-995a-4a02-a442-101399f8bdbc.png)
```suggestion
            <a href="#" class="logo"></a>
```